Tools/TestWebKitAPI/Tests/WebKit/SpacebarScrolling.cpp

```diff
@@ -49,6 +52,16 @@ static void didNotHandleKeyEventCallback(WKPageRef, WKNativeEventPtr event, cons
         didNotHandleKeyDownEvent = true;
 }
 
+
+static void didRunJavascript(WKSerializedScriptValueRef serializedScriptValue, WKErrorRef error, void* context)
+{
+    JSGlobalContextRef scriptContext = JSGlobalContextCreate(0);
```

Comment:
Nit - it looks like this leaks `scriptContext`. Do we need a `JSGlobalContextRelease`?